packages/theme-patternfly-org/scripts/md/parseMD.js

```diff
@@ -42,13 +42,13 @@ function toReactComponent(mdFilePath, source) {
       if (!frontmatter.id) {
         file.fail('id attribute is required in frontmatter for PatternFly docs');
       }
-      if (frontmatter.section === 'overview') {
-        // Temporarily override section until https://github.com/patternfly/patternfly-react/pull/4862 is in react-docs
-        // Affected pages are release notes and upgrade guides
-        frontmatter.section = 'developer-resources';
-      }
       source = frontmatter.source || source;
-      const slug = makeSlug(source, frontmatter.section, frontmatter.id);
+      let slug
+      if (frontmatter.subsection) {
+        slug = makeSlug({ source, section: frontmatter.section, subsection: frontmatter.subsection, id: frontmatter.id });
+      } else {
+        slug = makeSlug({ source, section: frontmatter.section, id: frontmatter.id });
+      }
```

Comment:
Change to `const slug = makeSlug({ source, section: frontmatter.section, subsection: frontmatter.subsection, id: frontmatter.id });`. If `frontmatter.subsection` is `undefined` that's okay.